pyvo/dal/ssa.py

```diff
@@ -696,10 +697,13 @@ def dateobs(self):
         observational data that went into the spectrum
         """
         dateobs = self.getbyutype("ssa:DataID.Date", decode=True)
-        if dateobs:
-            return Time(dateobs, format="iso")
-        else:
-            return None
+        try:
+            if not dateobs or np.isnan(dateobs):
+                return None
+        except TypeError:
+            # np.isnan can only check floats. If can't check for nan, pass it along
+            pass
```

Comment:
Looks like @bsipocz beat me to the punch before I could say anything! Thanks for the merge!

Some small context behind the try except on why I moved the check nan check to this try except, When testing I ran into this same error on stackoverflow: https://stackoverflow.com/questions/52657223/typeerror-ufunc-isnan-not-supported-for-the-input-types-and-the-inputs-could

Since the suggestion was to use `pandas` which isn't a current dependency for pyvo, I figured it was too much weight just for this small test to add a whole new dep. So the intention was to have `np.isnan` check the conditions it is able to, but to allow anything outside of those bounds to flow through to `Time` as before.

Reply:
This all looked good, the comment in the code seemed sufficient, and you addressed the review requests, too, so no more explanation is needed. 

And indeed, we would have asked you to write a workaround if you introduced pandas as a dependency here.